apbs/test/test_input_file/test_calculate/test_finite_element.py

```diff
@@ -0,0 +1,268 @@
+"""Test input file calculate.FiniteElement."""
+import logging
+import json
+import pytest
+from apbs.input_file.calculate.finite_element import FiniteElement
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+GOOD_TEMPLATE = {
+    "a priori refinement": None,
+    "boundary condition": None,
+    "calculate energy": True,
+    "calculate forces": True,
+    "charge discretization": None,
+    "domain length": [22.5, 19.5, 0.5],
+    "error based refinement": None,
+    "error tolerance": 1e-3,
+    "equation": None,
+    "ions": None,
+    "initial mesh resolution": 0.5,
+    "initial mesh vertices": 100000,
+    "maximum refinement iterations": 10,
+    "maximum vertices": 10000000,
+    "molecule": "foo",
+    "solute dielectric": 12,
+    "solvent dielectric": 78.54,
+    "solvent radius": 1.4,
+    "surface method": None,
+    "surface spline window": 0.3,
+    "temperature": 298.15,
+    "use maps": None,
+    "write atom potentials": "atom_potentials.txt",
+    "write maps": [
+        {"property": "potential", "format": "dx.gz", "path": "pot.dx.gz"}
+    ],
+}
+GOOD_A_PRIORI_REFINEMNT = ["geometric", "uniform"]
+GOOD_BOUNDARY_CONDITIONS = [
+    "zero",
+    "single sphere",
+    "multiple sphere",
+    "focus foo",
+]
+GOOD_BEER = ["White Bluffs FID"]
```

Comment:
Was this a test to see if I would find something or is this really used?

Reply:
@sobolevnrm I would have gone with "Atomic Ale B-17".

Reply:
It was a test.  I guess you passed. :)  Want me to remove it?

Reply:
I removed it so we look more, professional?